### DISCUSSION TITLE
support applescript automation?

### DISCUSSION THREAD

**Discussion by zzzqgrelax** [AvaloniaUI/Avalonia#11026] [Q&A]
i want to start my avalonia app from another vue app on MacOS
and i planned to use appleScript
how can it works

**Comment by maxkatz6** [comment]
If AppleScript works through NSAccessibility API, then it should work in 11.0 Avalonia. https://github.com/AvaloniaUI/Avalonia/blob/master/native/Avalonia.Native/src/OSX/automation.mm to see how automation is implemented in Avalonia for macOS.
